minimum-constraints-develop.txt

```diff
@@ -71,6 +74,7 @@ sphinxcontrib-serializinghtml==1.1.9; python_version >= '3.9'
 sphinxcontrib-websupport==1.2.4
 autodocsumm==0.2.12
 Babel==2.11.0
+roman-numerals-py==1.0.0
```

Comment:
roman-numerals-py 1.0.0 requires Python>=3.9, and is used by Sphinx>=8.2.0 which also requires Python>=3.9. Since the constraints are only used when the package is actually installed, it does not matter that the constraint has no python version marker. However, that prevents using the constraints file as a requirements file, and I think it would be more clear to add a python version marker: `; python_version >= '3.9'`.